cves/CVE-2013-0848.yml

```diff
@@ -131,7 +135,9 @@ lifetime:
           git log --stat abc..def -- your/file.c
           (where a)
       * the changelog between the two
-  answer:
+  answer: This piece of vulerable code seems as though it was neglected. A developer made a commit that to
```

Comment:
I think you may need to use `|` and move this part to the next line similar to other tags found in this file.